.gitignore

```diff
@@ -24,6 +24,8 @@ install/status.json
 custom/modules/Connectors/connectors/sources/ext/*/*/config.php
 # Logs files can safely be ignored.
 *.log
+# The silent upgrade scripts aren't needed.
+silentUpgrade*.php
```

Comment:
Why do you feel these aren't needed? the ```silentUpgrade.php``` file is used by the ```UpgradeWizard/``` module to execute upgrades to an existing instance through the CLI.